tsconfig.json

```diff
@@ -2,7 +2,9 @@
   "extends": "./node_modules/@hollowverse/common/tsconfig.json",
   "compilerOptions": {
     "maxNodeModuleJsDepth": 0,
-    "outDir": "./bin"
+    "outDir": "./bin",
+    "sourceMap": false
   },
-  "exclude": ["node_modules"]
+  "include": ["src"],
+  "exclude": ["node_modules", "bin", "**/*.snap"]
```

Comment:
We don't need to exclude "node_modules" and "**/*.snap".

Also, I'm not sure we should be checking in compiled files. We should compile on CI.

Reply:
Okay, I'll remove the `exclude` key and only generate and publish the binaries on the CI.